nodeServerless/Local CE SAM/CE_lambda_functions/card_get.js

```diff
@@ -0,0 +1,62 @@
+/**
+ *
+ * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @param {Object} event - API Gateway Lambda Proxy Input Format
+ *
+ * Context doc: https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html 
+ * @param {Object} context
+ *
+ * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+ * @returns {Object} object - API Gateway Lambda Proxy Output Format
+ * 
+ */
+
+let response;
+var AWS = require('aws-sdk');
+AWS.config.update({
+    region: "us-west-2"
+})
+
+let docClient = new AWS.DynamoDB.DocumentClient();
+// let ddb = new AWS.DynamoDB();
+
+exports.lambdaHandler = async (event, context) => {
+    try {
+        console.log("\n\n*********************\n");
+        var id = event.queryStringParameters.scryfallid;
```

Comment:
Maybe im missing some context, but query parameters are the wrong way to encode these. These should probably be encoded in a request body or uri path.

Query strings are for optional flags like query filters. Webapps and automatic webcaching systems treat these differently which is why it's important to follow the convention.

Reply:
I thought that search parameters did go in the query string? I'm looking at this stackoverflow right now https://stackoverflow.com/a/45462793. What's the convention?